src/v/pandaproxy/schema_registry/json.cc

```diff
@@ -1433,6 +1433,32 @@ bool is_superset(
     return true;
 }
 
+void sort(json::Value& val) {
+    switch (val.GetType()) {
+    case rapidjson::Type::kFalseType:
+    case rapidjson::Type::kNullType:
+    case rapidjson::Type::kNumberType:
+    case rapidjson::Type::kStringType:
+    case rapidjson::Type::kTrueType:
+        break;
+    case rapidjson::Type::kArrayType: {
+        for (auto& v : val.GetArray()) {
+            sort(v);
+        }
+        break;
+    }
```

Comment:
some arrays should be sorted i think. looking at draft7
```
        "allOf": { "$ref": "#/definitions/schemaArray" },
        "anyOf": { "$ref": "#/definitions/schemaArray" },
        "oneOf": { "$ref": "#/definitions/schemaArray" },
        "required": { "$ref": "#/definitions/stringArray" },
                "enum": {
            "type": "array",
            "items": true,
            "minItems": 1,
            "uniqueItems": true
        },
                "type": {
            "anyOf": [
                { "$ref": "#/definitions/simpleTypes" },
                {
                    "type": "array",
                    "items": { "$ref": "#/definitions/simpleTypes" },
                    "minItems": 1,
                    "uniqueItems": true
                }
            ]
        },
```

(note that "items" should not be sorted)

Reply:
This'll have to wait.